### ISSUE TITLE
Not working ( explanation in the description )

### ISSUE THREAD

**Issue by kanlukasz** [hagishi/vscode-php-xdebug#2]
It does not work for me or I set something wrong.
I do not know what the problem is. 

#### VS Code:
```
Version: 1.35.0-insider
Commit: 553cfb2c2205db5f15f3ee8395bbd5cf066d357d
Date: 2019-06-04T00:13:30.572Z
Electron: 3.1.8
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
```
#### xDebug:
![obraz](https://user-images.githubusercontent.com/30685349/58955791-21e85400-879d-11e9-8d09-292f98ae20ca.png)

#### PHP (installed with XAMPP):
![obraz](https://user-images.githubusercontent.com/30685349/58955846-39bfd800-879d-11e9-934a-4195401ba13d.png)

#### Problems that i see:
- there is no config:
![obraz](https://user-images.githubusercontent.com/30685349/58955930-74297500-879d-11e9-86be-6d6f56ac171d.png)
- if i use `Listen for XDebug`:
![obraz](https://user-images.githubusercontent.com/30685349/58956002-9b804200-879d-11e9-84ea-97cf01fce365.png)

#### Note that:
- i think PHP works right because VS Code linter works without problem

Could you help me with this?

**Comment by hagishi** [comment]
Fixed issue#2 with v0.0.7

**Comment by kanlukasz** [comment]
It work. Even **without** configuration and **without** open folder. This is **VERY** good. This addon have a great future. Keep going

![obraz](https://user-images.githubusercontent.com/30685349/59010228-d7181c00-8830-11e9-818f-cbbf6759d389.png)

_By the way - if can i suggest something - it would be good to create more specifically readme file of this repository. If you don't have time for this, I will try to propose something in my free time._
